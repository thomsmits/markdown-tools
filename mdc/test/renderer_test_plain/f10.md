# Chapter
## Slide 6.3

Text with a [one][1] and another [two][2]

* [three][3]
* [four][4]

> Quote [five][5]

>! Important [six][6]

>? Question [seven][7]

>? Question [eight][8]

[1]: https://en.wikipedia.org/wiki/Hobbit#Lifestyle
[2]: https://en.wikipedia.org/wiki/Hobbit#Lifestyle "Hobbit lifestyles"
[3]: https://en.wikipedia.org/wiki/Hobbit#Lifestyle 'Hobbit lifestyles'
[4]: https://en.wikipedia.org/wiki/Hobbit#Lifestyle (Hobbit lifestyles)
[5]: <https://en.wikipedia.org/wiki/Hobbit#Lifestyle> "Hobbit lifestyles"
[6]: <https://en.wikipedia.org/wiki/Hobbit#Lifestyle> 'Hobbit lifestyles'
[7]: <https://en.wikipedia.org/wiki/Hobbit#Lifestyle> (Hobbit lifestyles)
[8]: <https://en.wikipedia.org/wiki/Hobbit#Lifestyle>
