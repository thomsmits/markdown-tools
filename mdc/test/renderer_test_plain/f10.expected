# Chapter
## Slide 6.3

Text with a [one](https://en.wikipedia.org/wiki/Hobbit#Lifestyle) and another [two](https://en.wikipedia.org/wiki/Hobbit#Lifestyle "Hobbit lifestyles")

* [three](https://en.wikipedia.org/wiki/Hobbit#Lifestyle "Hobbit lifestyles")
* [four](https://en.wikipedia.org/wiki/Hobbit#Lifestyle "Hobbit lifestyles")

> Quote [five](https://en.wikipedia.org/wiki/Hobbit#Lifestyle "Hobbit lifestyles")

>! Important [six](https://en.wikipedia.org/wiki/Hobbit#Lifestyle "Hobbit lifestyles")

>? Question [seven](https://en.wikipedia.org/wiki/Hobbit#Lifestyle "Hobbit lifestyles")

>? Question [eight](https://en.wikipedia.org/wiki/Hobbit#Lifestyle)
